### PR TITLE
Changelog documentation

### LINKED ISSUE #71: maybe add a list of the open source packages we rely on
just to shout out to what we're building on

### PR DESCRIPTION
**Pull request by lboeman**
Adds `CHANGELOG.md` and links to it from the dashboard. Currently contains major features added since alpha. @wholmgren I know we want a dashboard user oriented changelog, and that's what I've aimed for here. Do you think we should maintain a separate code repository changelog in this file, and an HTML changelog of dashboard features and links to documentation

also closes #71

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Changelog
+
+All notable changes to the Solar Forecast Arbiter Dashboard will be documented
```
Comment (wholmgren) [inline CHANGELOG.md]: This implies to me that this just tracks the Dashboard, but it's really anything that a dashboard user would encounter.

We should also encourage users to read the [`solarforecastarbiter-core` what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+- Report form now includes all deterministic metrics options identified by
+  stakeholders. Also includes options to calculate metrics by categories Total,
+  Year, Month, Date, and Hour of Day.
+
+- Ability to compare Forecasts to Aggregates in reports.
+
+- Report's may be downloaded in HTML format at
+  `/reports/<report_id>/downloads/html`. Note that there are no currently
+  functioning links to this endpoint.
```
Comment (alorenzo175) [inline CHANGELOG.md]: There is a link in newer reports.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+### Added
+
+- Report form now includes all deterministic metrics options identified by
+  stakeholders. Also includes options to calculate metrics by categories Total,
+  Year, Month, Date, and Hour of Day.
+
+- Ability to compare Forecasts to Aggregates in reports.
+
+- Report's may be downloaded in HTML format at
```
Comment (wholmgren) [inline CHANGELOG.md]:
```suggestion
- Reports may be downloaded in HTML format at
```

File: CHANGELOG.md (hunk 2 above)
Comment (wholmgren) [inline CHANGELOG.md]: I don't understand the last sentence

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Changelog
+
+All notable changes to the Solar Forecast Arbiter Framework will be documented
+in this file.
+
+Detailed changes to the Solar Forecast Arbiter Core python library can be found
+in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
+
+## [Unreleased]
```
Comment (alorenzo175) [inline CHANGELOG.md]: might as well change to 1.0beta3 and today's date

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+### Added
+
+- Report form now includes all deterministic metrics options identified by
+  stakeholders. Also includes options to calculate metrics by categories Total,
```
Comment (wholmgren) [inline CHANGELOG.md]: Split into another `-` item. "Report form now includes options..."

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+- Report's may be downloaded in HTML format at
+  `/reports/<report_id>/downloads/html`. Note that there are no currently
+  functioning links to this endpoint.
+
+- Report downloads contain a GPG signed report as well as md5, sha1 and sha256
+  checksums for validation.
+
+- CHANGELOG.md (this file) for tracking and communicating changes.
+
+
```
Comment (alorenzo175) [inline CHANGELOG.md]: is https://github.com/SolarArbiter/solarforecastarbiter-api/commit/8f66fe45beda692dd4d4d92bf82c6f5b14173051 worth adding?

File: README.md [hunk 7]
```diff
@@ -1,57 +1,65 @@
...
+- [Requests](https://requests.readthedocs.io/en/master/)
+- [Python-JOSE](https://python-jose.readthedocs.io/en/latest/)
+- [Blinker](https://pythonhosted.org/blinker/)
+- [Cryptography](https://cryptography.io/en/latest/)
+- [PyTables](https://www.pytables.org/usersguide/tutorials.html)
+- [Pandas](https://pandas.pydata.org/)
+
+Authenication services for the Solar Forecast Arbiter by provided by [Auth0](https://auth0.com/)
+
+Centralized logging for the Solar Forecast Arbiter framework provided by
```
Comment (alorenzo175) [inline README.md]:
```suggestion
Centralized error reporting for the Solar Forecast Arbiter framework is provided by
```

File: sfa_dash/templates/navbar.html [hunk 8]
```diff
@@ -12,7 +12,7 @@ <h1>Solar Forecast Arbiter</h1>
     <div class="collapse navbar-collapse" id="navbarCollapse">
       <ul class="navbar-nav mr-auto">
         <li class="nav-item">
-            <a class="nav-link" href="{{ url_for('changelog') }}">Changelog</a>
+            <a class="nav-link" href="https://https://github.com/SolarArbiter/solarforecastarbiter_dashboard/blob/master/CHANGELOG.md">Changelog</a>
```
Comment (alorenzo175) [inline sfa_dash/templates/navbar.html]:
```suggestion
            <a class="nav-link" href="https://github.com/SolarArbiter/solarforecastarbiter_dashboard/blob/master/CHANGELOG.md">Changelog</a>
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+- Permissions acting on aggregates are now accessible via a Role's permission
+  listing.
+
+
+## [1.0beta2] - 2019-11-18
+
+### Added
+- Aggregates can be created through the dashboard. See  [Aggregate Documentation](https://solarforecastarbiter.org/documentation/dashboard/working-with-data/#create-new-aggregate) 
+
```
[the remark this answers is not in the document]
Comment (alorenzo175) [reply CHANGELOG.md]:
```suggestion

- Day-ahead probabilistic reference forecasts based on the GEFS are available for DOE RTC, NOAA SURFRAD, NOAA SOLRAD, and NREL MIDC networks.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+### Added
+
+- Report form now includes all deterministic metrics options identified by
+  stakeholders.
+
+- Report form now includes options to calculate metrics by categories Total,
+  Year, Month, Date, and Hour of Day.
+
+- Ability to compare Forecasts to Aggregates in reports.
```
Comment (wholmgren) [inline CHANGELOG.md]:
```suggestion
- Ability to analyze forecasts of aggregated observations in reports.
```

Comment (wholmgren) [reply CHANGELOG.md]: previous text assumed too much familiarity with data model

File: CHANGELOG.md (hunk 6 above)
Comment (wholmgren) [reply CHANGELOG.md]: sure

**Pull request merged**